schema.json

```diff
@@ -196,10 +196,14 @@
       "type": "object",
       "properties": {
         "lower": {
-          "type": "integer"
```

Comment:
Likewise here and below.

Reply:
Done in https://github.com/OpenLightingProject/pid-schema/pull/6/commits/e175ca69a08581d13dcdae5727ec0f11106c8214